database/schema.sql

```diff
@@ -0,0 +1,15 @@
+    DROP DATABASE IF EXISTS slideShowData;
+    CREATE DATABASE slideShowData;
+    USE slideShowData;
+    
+    
+        
+    CREATE TABLE products (
+        id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        productId INT,
+        productName varchar(255),
+        productDescription varchar (255),
```

Comment:
i wouldn't limit this length to 255, maybe use TEXT instead?

Reply:
good idea! thanks!